src/Transport/RestTransport.php

```diff
@@ -91,10 +97,21 @@ public static function build($apiEndpoint, $restConfigPath, array $config = [])
         $config += [
             'httpHandler'  => null,
             'clientCertSource' => null,
+            'interceptors' => []
         ];
         list($baseUri, $port) = self::normalizeServiceAddress($apiEndpoint);
         $requestBuilder = new RequestBuilder("$baseUri:$port", $restConfigPath);
-        $httpHandler = $config['httpHandler'] ?: self::buildHttpHandlerAsync();
+        $interceptors = $config['interceptors'];
+        if (!$config['httpHandler'] && !empty($interceptors)) {
+            $stack = HandlerStack::create();
+            foreach ($interceptors as $interceptor) {
+                $stack->push($interceptor);
+            }
+            $client = new Client(['handler' => $stack]);
+            $httpHandler = self::buildHttpHandlerAsync($client);
+        } else {
+            $httpHandler = $config['httpHandler'] ?: self::buildHttpHandlerAsync();
```

Comment:
```suggestion
            $httpHandler = $config['httpHandler'] ? $config['httpHandler'] : self::buildHttpHandlerAsync();
```
I think this is missing the `true` return value, unless there is some magic that actually assumes the true case returns the result of the ternary expression.